drivers/uart_seds/uart_sedsconfiguration.asn

```diff
@@ -0,0 +1,15 @@
+-- DRIVER CONFIGURATION MODEL [user-editable]
+-- Shall contain definition of Driver Configuration.
+-- This file will be used by ocarina-components.aadl and Board1.xml.
+UART-SEDS-DRIVER-CONFIGURATION DEFINITIONS ::= BEGIN
+
+    UART-SEDS-Device-T    ::= ENUMERATED {uart0, uart1, uart2, uart3, uart4}
+
+    UART-SEDS-Baudrate-T  ::= ENUMERATED {b9600, b19200, b38400, b57600, b115200, b230400}
```

Comment:
Some speeds are not supported by our device

Reply:
done